packages/libs/pactjs-generator/src/contract/generation/tests/__snapshots__/generator.test.ts.snap

```diff
@@ -203,11 +196,10 @@ declare module '@kadena/client' {
 
 exports[`generateDts uses the property type if there is no mapped value for that 1`] = `
 "
-import type { ICapabilityItem } from '@kadena/client';
-import type { IPactDecimal, IPactInt, PactReference } from '@kadena/types';
+import type { IPactDecimal, IPactInt, PactReference, ICap } from '@kadena/types';
 
 interface ICapability_Coin_GAS {
-  (name: 'coin.GAS'): ICapabilityItem;
+  (name: 'coin.GAS'): ICap;
```

Comment:
Nitpick: seeing all these renames making a lot of sense, except this one is the only shortened.

Reply:
Yes, we moved away from just wrapping the `ICap` from `@kadena/types` as `ICapabilityItem` and chose to use the `ICap` directly